src/lib.rs

```diff
@@ -83,13 +83,14 @@
 //!   have its own identifying number unique to the font, its ID.
 extern crate arrayvec;
 extern crate stb_truetype;
-extern crate linked_hash_map;
 
 mod geometry;
 mod rasterizer;
 
+#[cfg(feature = "gpu_cache")]
```

Comment:
Should mod `support` (or `support::bst`) be moved into `gpu_cache`?
I'm not sure whether this mod would be used from other mod.